src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $card-size);
+  gap: 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-size);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-size);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-size);
+  }
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $card-size;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
```

Comment:
Doesn't work correctly when the width less than 488px
![image](https://user-images.githubusercontent.com/119349933/208269588-42f48059-cc1c-43d4-b27b-a5488637c409.png)

Reply:
thanks for your comments! already fixed them